src/test/resources/taxInfo.json

```diff
@@ -1,9 +1,17 @@
 {
-    "amount": 900,
-    "portions": [
-        {
-            "name": "VAT 10%",
-            "amount": 900
+    "value": {
+        "currency": "USD",
+        "amount": "900",
+        "country": "US"
+    },
+  "portions": [
```

Comment:
alignment/formatting seems wrong ... activate IDE auto-formatting?

Reply:
If I format it via IDE then I get a ton of changes (not sure why).